raiden/tests/scenarios/scenario-funding channel.yaml

```diff
@@ -0,0 +1,256 @@
+
+      - serial:
+          name: Scenario F9 (Depositing 0 tokens)
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000} ## we expect this to work, right?
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F10 (Depositing negative amount I)
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: -1, expected_http_status: 409}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F11 (Depositing negative amount II)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"} ## This seems to be a bug. When a node successfully deposits 98_000_000_000_000_000_000 (see code 200), it's status should not be 99_000_000_000_000_000_000
+            - deposit: {from: 0, to: 1, total_deposit: 98_000_000_000_000_000_000, expected_http_status: 409}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"} ## This seems to be a bug. When a node successfully deposits 98_000_000_000_000_000_000 (see code 200), it's status should not be 99_000_000_000_000_000_000
+
+      - serial:
+          name: Scenario F12 (Depositing incredibly huge number)
+          tasks:
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+           - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000, expected_http_status: 402} ## !! the problem is, the node does not have that amount of tokens so it would return "not sufficient funds"
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F13 (Depositing smallest possible number)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, expected_http_status: 200}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+
+#      - serial:
+#          name: Scenario F14 (Depositing more than channel limit of 100 - with ETH it would be 0.15 ETH)
+#          tasks:
+#            - deposit: {from: 0, to: 1, total_deposit: 100} ## This can not be tested with this token
```

Comment:
Pls create a follow-up issue for this case after this gets merged.

Reply:
ok